pkg/controller/dataimportcron-controller.go

```diff
@@ -1235,7 +1238,8 @@ func (r *DataImportCronReconciler) setJobCommon(cron *cdiv1.DataImportCron, obj
 	}
 	util.SetRecommendedLabels(obj, r.installerLabels, common.CDIControllerName)
 	labels := obj.GetLabels()
-	labels[common.DataImportCronLabel] = getCronJobLabelValue(cron.Namespace, cron.Name)
+	labels[common.DataImportCronNsLabel] = cron.Namespace
```

Comment:
Are we worried about upgrade-carried resources not having this on them? maybe worth a small unit test that an existing cronjob without this label gets it on update?

Reply:
In upgrade it will set this label (assuming the DIC exists). Can sure add it to the unit tests.